.github/workflows/bumpdevversion.yml

```diff
@@ -0,0 +1,40 @@
+on:
+  pull_request:
+    branches:
+      - devel
+  # Schedule runs on 9am every Sunday
+  schedule:
+    - cron: '0 9 * * 0'
+
+jobs:
+  bump-dev-version:
+    name: Bump GMT development version
+    runs-on: ubuntu-latest
+
+    steps:
+      # Checkout current git repository
+      - name: Checkout
+        uses: actions/checkout@v2.3.1
+
+      # Change Github sha hash
+      - name: Update github_rev
+        uses: mikefarah/yq@3.3.2
+        with:
+          cmd: "yq write --inplace recipe/meta.yaml 'source.github_rev' 123456789"
+
+      # Bump build version number
+      - name: Bump build number
+        uses: mikefarah/yq@3.3.2
+        with:
+          cmd: "yq write --inplace recipe/meta.yaml 'build.number' 1"
```

Comment:
[yq](https://github.com/mikefarah/yq) (YAML processor) example adapted from https://github.com/tylerauerbeck/open-management-portal-deployment/blob/82796429c2fb9226df6e76e41c4fccb4bb012046/.github/workflows/release.yml.

Seems like the first line `{% set version = "6.2.0dev" %}` isn't a standard YAML thing and so the file can't be parsed?